examples/pastebin/.gitignore

```diff
@@ -0,0 +1 @@
+/upload/
```

Comment:
I want to keep this folder around so that user's don't have to create it manually.